README.md

```diff
@@ -95,7 +95,7 @@ function setupWebViewJavascriptBridge(callback) {
 	window.WVJBCallbacks = [callback];
 	var WVJBIframe = document.createElement('iframe');
 	WVJBIframe.style.display = 'none';
-	WVJBIframe.src = 'https://__bridge_loaded__';
+	WVJBIframe.src = 'https://wvjb-bridge_loaded';
```

Comment:
There's still an underscore in there. Are you sure this works as expected?

Reply:
I think there is a type error. It should be `wvjb-bridge-loaded`, is that right? @nilswiersema